pyproject.toml

```diff
@@ -22,6 +22,85 @@ ignore_errors = true
 fail_under = 90.0
 show_missing = true
 
+[tool.poetry]
+name = "sceptre"
+version = "3.1.0"
+description = "Cloud Provisioning Tool"
+authors = ["Cloudreach <sceptre@cloudreach.com>"]
+maintainers = []
+license = "Apache-2.0"
+readme = "README.md"
+homepage = "https://docs.sceptre-project.org/"
+repository = "https://github.com/Sceptre/sceptre"
+documentation = "https://docs.sceptre-project.org/"
+keywords = ["sceptre", "aws", "cloud", "provisioning", "cloudformation"]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Developers",
+  "Natural Language :: English",
+  "Environment :: Console",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+]
+packages = [
+  { include = "sceptre/" },
+]
+include = ["CHANGELOG.md", "README.md"]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+boto3 = "^1.3.0"
+click = "^8.1.3"
+colorama = "^0.4.4"
+cfn-flip = "^1.3.0"
+deepdiff = "^5.8.1"
+Jinja2 = "^3.1.2"
+jsonschema = "^4.6.0"
+networkx = ">=2.4,<2.6"
+PyYAML = "^6.0"
+sceptre-cmd-resolver = "^1.2.1"
+sceptre-file-resolver = "^1.0.6"
+six = "^1.16.0"
+troposphere = "^4.0.2"
+
+[tool.poetry.dev-dependencies]
+behave = "^1.2.6"
+coverage = {extras = ["toml"], version = "^6.4.1"}
+flake8 = "^4.0.1"
+pre-commit = "^2.19.0"
+freezegun = "^1.2.1"
+moto = "^3.1.12"
+Pygments = "^2.12.0"
+pytest = "^7.1.2"
+pytest-cov = "^3.0.0"
+pytest-sugar = "^0.9.4"
+tox = "^3.25.0"
+Sphinx = "<5.0"
+sphinx-click = "^4.1.0"
+sphinx-rtd-theme = "^1.0.0"
+sphinx-autodoc-typehints = "<1.18"
+
+[tool.poetry.scripts]
+sceptre = "sceptre.cli:cli"
+
+[tool.poetry.plugins."sceptre.hooks"]
```

Comment:
What's the point of the quotes here?

Reply:
You know, I asked myself the very same question when I was copying this over from the original PR I had 😆 

I know I had a reason for them previously, but I couldn't tell you what that is now. My guess would be, something didn't work right without them. But I can do some experimentation to figure that out.